### DISCUSSION TITLE
1.5.11 release date

### DISCUSSION THREAD

**Discussion by b005t3r** [bytedeco/javacpp#778] [General]
Is there a rough javacpp 1.5.11 release date maybe?

It's currently not possible to process HDR videos properly with 1.5.11 since there's no ~~zlib~~ zscale support enabled in the embedded ffmpeg version, but 1.5.11's ffmpeg build will have ~~zlib~~ zcale enabled, which should make working with HDR videos finally possible.

**Comment by saudet** [comment]
I didn't do anything about zlib, that should be in 1.5.10

**Reply by b005t3r** [reply]
Sorry, I meant zscale not zlib. I'm talking about this change:
https://github.com/bytedeco/javacpp-presets/commit/811d17355750752c374a80841026fcf365a5ba34

**Comment by b005t3r** [comment]
@saudet could you tell me if the 1.5.11 release is planned anytime soon?

**Reply by saudet** [reply]
Probably before the end of the year?
